_input/listbox-button/index.html

```diff
@@ -185,7 +185,282 @@ <h3>Selected</h3>
         </div>
     </div>
 </div>
-
 <hr />
+<h2>Example 3: Listbox with group headers</h2>
+<p>
+    ARROW keys move the active descendant <em>and</em> automatically updated the selected state. 
```

Comment:
> <em>and</em> automatically updated the selected state
only apply for the second example in this section, right? So I'd recommend putting that note after the 

```
<h3> Selected, Automatic Selection </h3>
```